msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -5,62 +5,59 @@ description: Identify the meeting apps API references with examples and Code sam
-|**NotificationSignal** | `activity.TeamsNotifyUser(true, "https://teams.microsoft.com/l/bubble/APP_ID?url=&height=&width=&title=<title>&completionBotId=BOT_APP_ID");` |
-|**Meeting Details** | `TeamsMeetingInfo (string id = default);` |
+|Method| Description| Source|
+|---|---|----|
+|[**Get user context**](#getusercontext-api)| Get contextual information to display relevant content in a Teams tab.| Microsoft Teams client SDK|
+|[**Get participant**](#getparticipant-api)| Fetch participant information by meeting ID and participant ID. |Microsoft Bot Framework SDK|
+|[**Notification signal**](#notificationsignal-api)| Provide meeting signals using the existing conversation notification API for user-bot chat and allows to notify user action that shows an in-meeting dialog box. |Microsoft Bot Framework SDK|
+|[**Meeting details**](#meeting-details-api)| Get static meeting metadata. |Bot SDK |
+|[**Share app content to stage**](#shareappcontenttostage-api)| Share specific parts of the app to meeting stage from the app side panel in a meeting. |Microsoft Teams client SDK|
+|[**Get app content stage sharing state**](#getappcontentstagesharingstate-api)| Fetch information about apps' sharing state on the meeting stage. |Microsoft Teams client SDK|
+|[**Get app content stage sharing capabilities**](#getappcontentstagesharingcapabilities-api)| Fetch the apps' capabilities for sharing to the meeting stage. |Microsoft Teams client SDK|
+|[**Real-time Teams meeting events**](#real-time-teams-meeting-events)|Fetch real-time meeting events, such as actual start and end time.| Bot SDK|
 
 ## GetUserContext API
 
-To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab when running in the meeting context and is added for the response payload.
+To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab running in the meeting context and is added for the response payload.
 
 ## GetParticipant API
 
+Allows a bot to fetch participant information by meeting ID and participant ID. The API includes query parameters, examples, and response codes.
+
 > [!NOTE]
 > * Do not cache participant roles since the meeting organizer can change the roles any time.
-> * Teams does not currently support large distribution lists or roster sizes of more than 350 participants for the `GetParticipant` API.
-
-The `GetParticipant` API allows a bot to fetch participant information by meeting ID and participant ID. The API includes query parameters, examples, and response codes.
+> * Currently, limitation to distribution lists or roster sizes are less than 350 participants for the `GetParticipant` API.
```

Comment:
Currently, this API is not supported for distribution lists or meetings whose roster size is greater than 350 participants.

Reply:
Done